Procfile

```diff
@@ -0,0 +1,3 @@
+app: lapis server
+db: postgres -D /usr/local/var/postgres
```

Comment:
This is the default mac location. I don't know what it is on other systems. I'm happy to make the $POSTGRES_DATA_DIR or something else if there's a different directly so this is usable by anyone.

Reply:
Not in Linux. This is what `SHOW data_directory;` spits out in the server:

`/var/lib/postgresql/9.5/main`